Doc/library/tempfile.rst

```diff
@@ -87,6 +87,9 @@ The module defines the following user-callable items:
    attribute is the underlying true file object. This file-like object can
    be used in a :keyword:`with` statement, just like a normal file.
 
+   On POSIX (only), a process that is terminated abruptly with SIGKILL
+   cannot automatically delete :func:`NamedTemporaryFile`s it created.
```

Comment:
As noted by the GitHub action, this reStructuredText is broken.

Suggested alternate:

~~~~~~ text
   On POSIX (only), a process that is terminated abruptly with SIGKILL
   cannot automatically delete a :func:`NamedTemporaryFile` it created.
~~~~~~

Reply:
oop, pushed a different resolution before seeing your suggestion...

Thanks & sorry, would have caught this but `make html` is failing for me and I foolishly thought I could get by without it...

Reply:
No worries.  While I like reStructuredText better than the some widely used alternatives... it has warts.